hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -1022,22 +1020,6 @@ public void update(HoodieRollbackMetadata rollbackMetadata, String instantTime)
     }
   }
 
-  protected void validateRollback(
```

Comment:
Why should this be relaxed?  Does not look safe for Spark.

Reply:
this change need for compation with spark , otherwise would report error @yihua

Reply:
> Why should this be relaxed? Does not look safe for Spark.

It should be safe now, because the MDT log reader can filter out invalid commits based on DT timeline, we need to do this, because Flink has relax the compaction now, if people uses Spark to write into Flink table, then the error throws.

Another choice is to disable the MDT for Flink again, I don't think MDT is production ready for long running MOR job.